sources/packages/forms/src/form-definitions/studentprofile.json

```diff
@@ -3449,6 +3449,424 @@
+          "persistent": true,
+          "inputFormat": "html",
+          "protected": false,
+          "dbIndex": false,
+          "case": "",
+          "truncateMultipleSpaces": false,
+          "encrypted": false,
+          "redrawOn": "",
+          "clearOnHide": true,
+          "customDefaultValue": "",
+          "calculateValue": "",
+          "calculateServer": false,
+          "allowCalculateOverride": false,
+          "validateOn": "change",
+          "validate": {
+            "required": false,
+            "minLength": "",
+            "maxLength": "",
+            "minWords": "",
+            "maxWords": "",
+            "pattern": "",
+            "customMessage": "",
+            "custom": "",
+            "customPrivate": false,
+            "json": "",
+            "strictDateValidation": false,
+            "multiple": false,
+            "unique": false
+          },
+          "unique": false,
+          "errorLabel": "",
+          "errors": "",
+          "key": "textArea",
+          "tags": [],
+          "properties": {},
+          "conditional": {
+            "show": null,
+            "when": null,
+            "eq": "",
+            "json": ""
+          },
+          "customConditional": "",
+          "logic": [],
+          "attributes": {
+            "style": "max-height: 200px; overflow: auto"
```

Comment:
pls add custom classes in our project and use them here

Reply:
For this PR, I did not make any change on the style.

Reply:
@ann-aot I guess that it is following the same way that it was done for the designation agreement consent text (form.io `designationagreementdetails`).
I believe that we made an exception for that case.

Reply:
okay